Tests/NIOTestUtilsTests/ByteToMessageDecoderVerifierTest.swift

```diff
@@ -0,0 +1,167 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2019 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@testable import NIO
+import NIOTestUtils
+import XCTest
+
+typealias VerificationError = ByteToMessageDecoderVerifier.VerificationError<String>
+
+class UtilitiesTest: XCTestCase {
```

Comment:
Let's name this `ByteToMessageDecoderVerifierTest` instead.

Reply:
oopsie, copy & pasta for the win